packages/teams-js/src/public/runtime.ts

```diff
@@ -202,7 +204,7 @@ export const teamsRuntimeConfig: Runtime = {
       fullTrust: {},
     },
     teamsCore: {},
-    video: {},
+    video: { mediaStream: true },
```

Comment:
T2.1 should be using the hub SDK and should pass down a runtime value. To be verified, but that's the current plan.

Reply:
Confirmed, it will be added by this PR: https://office.visualstudio.com/ISS/_git/metaos-hub-sdk/pullrequest/1635202?path=/packages/metaos-hub-sdk/src/Services/MessageHandling/app-message-handler.ts&_a=files